readme.md

```diff
@@ -57,6 +57,12 @@ Any arguments to pass through to `fn`.
 
 Support for passing arguments on to the `fn` is provided in order to be able to avoid creating unnecessary closures. You probably don't need this optimization unless you're pushing a *lot* of functions.
 
+### limit.map(list, fn, ...args)
```

Comment:
Not sure we need the `...args`. People can just pass it to the `fn`.

Reply:
That's usually what I do, but I assumed you'd want consistency with the main function which does take args.

Maybe for a map function it makes more sense to truncate the parameters.